src/pages/Admin/AdminUpdateClass.tsx

```diff
@@ -1,7 +1,214 @@
+  background: none;
+  color: ${({ theme }) => theme.color.primary};
+  width: 120px;
+  height: 40px;
+  border-radius: 10px;
+  font-weight: 400;
+  cursor: pointer;
+  border-color: white;
+  &:hover {
+    border: none;
+    background-color: ${({ theme }) => theme.background.yellow};
+    color: ${({ theme }) => theme.color.primary};
+  }
+`;
+
+const iconStyles = {
+  fontSize: `1.3rem`,
+  opacity: `50%`,
+  cursor: `pointer`,
+  transition: `all ease 0.1s`,
+};
+
+const EditClassIcon = styled(RiEditBoxFill)`
+  ${iconStyles}
+  margin-right: 20px;
+  color: ${({ theme }) => theme.color.yellow};
+  font-size: 2rem;
+  &:hover {
+    opacity: 100%;
+  }
+`;
+const EditClassContentIcon = styled(AiOutlineEdit)`
+  ${iconStyles}
+  color: ${({ theme }) => theme.color.yellow};
+  &:hover {
+    opacity: 100%;
+  }
+`;
+const TrashIcon = styled(BiTrash)`
+  ${iconStyles}
+  margin-left: 10px;
+  margin-right: 10px;
+  color: ${({ theme }) => theme.color.red};
+  &:hover {
+    opacity: 100%;
+  }
+`;
+const DateIcon = styled(BsCalendarDate)`
+  ${iconStyles}
+  opacity: 100%;
+  color: ${({ theme }) => theme.color.primary};
+`;
+const AddClassContentIcon = styled(MdAdd)`
+  ${iconStyles}
+  color: ${({ theme }) => theme.color.yellow};
+  &:hover {
+    opacity: 100%;
+  }
+`;
+const Items = styled.ul`
+  margin-left: 50px;
+  margin-top: 10px;
+  font-weight: 100;
+`;
+const Item = styled.li`
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+`;
+const AddClassWrapper = styled.section`
+  display: flex;
+  gap: 10px;
+`;
+const Form = styled.form`
+  display: flex;
+  gap: 10px;
+`;
 
 const AdminUpdateClass = () => {
-  return <div>Admin update class</div>;
+  return (
+    <SyllabusContainer>
+      <SyllabusHeadlineWrapper>
+        <SyllabusHeadline>
+          Syllabus
+          <SyllabusDate>(March 1st, 2022 - December 1st, 2022)</SyllabusDate>
+          <DateIcon />
+        </SyllabusHeadline>
+        <AddClassButton type="button">+ New Class</AddClassButton>
+      </SyllabusHeadlineWrapper>
+      <Line />
+      <Classes>
+        <Class>
+          <EditClassIcon />
+          <ClassContent>
+            <article>
+              <ClassDate>March,10th,2022</ClassDate>
+              <ClassTitleWrapper>
+                <ClassTitle>Class 1: Introduction</ClassTitle>
+                <TrashIcon />
+              </ClassTitleWrapper>
+              <ClassDescription>
+                In this course, you will learn about JavaScript data types,
+                built-in methods, and variables.
+              </ClassDescription>
+            </article>
+            <Items>
+              <Item>
+                <EditClassContentIcon />
+                <TrashIcon />
+                <span>Stream</span>
+              </Item>
+              <AddClassWrapper>
+                <AddClassContentIcon />
+                <Form>
+                  <label htmlFor="class-item">
+                    <select>
```

Comment:
I think we better not use select/option, since it cannot be styled.

> There are only a few style attributes that can be applied to an `<option>` element.
>
> This is because this type of element is an example of a "replaced element". They are OS-dependent and are not part of the HTML/browser. It cannot be styled via CSS.
>
>There are replacement plug-ins/libraries that look like a `<select>` but are actually composed of regular HTML elements that CAN be styled.   

https://stackoverflow.com/questions/7208786/how-to-style-the-option-of-an-html-select-element

We can replace it with div for example.